src/index.ts

```diff
@@ -4,17 +4,13 @@ import { middleware } from "./middleware/middleware";
 import { sign } from "./sign/index";
 import { verify } from "./verify/index";
 import { verifyAndReceive } from "./middleware/verify-and-receive";
-import {
-  EventHandlerOptions,
-  EventState,
-  WebhookEvent as WebhookEventOptions,
-} from "./types";
+import { EventHandlerOptions, EventState, WebhookEvent } from "./types";
 import { EventNames } from "./generated/types";
 import { GetWebhookPayloadTypeFromEvent } from "./generated/api";
 import { IncomingMessage, ServerResponse } from "http";
 
 class Webhooks {
-  public sign: (secret: string, payload: string | object) => string;
+  public sign: (payload: string | object) => string;
```

Comment:
this was actually incorrect, the `secret` is set in the internal state of the `webhooks` instance. You only need to set the `secret` parameter when you use the standalone `sign` method